tox.ini

```diff
@@ -21,8 +21,9 @@ commands = coverage erase
            coverage run --include=charmtools/* {envbindir}/nosetests {posargs}
            coverage report
 deps =
-    {toxinidir}
+    -e{toxinidir}
     -r{toxinidir}/requirements.txt
+    ipdb
```

Comment:
I didn't mean to commit these, but they make debugging tests nicer, so I'm leaving it in as a drive-by.